cmd/tailscaled/tailscaled.service

```diff
@@ -20,5 +20,17 @@ CacheDirectory=tailscale
 CacheDirectoryMode=0750
 Type=notify
 
+LockPersonality=true
+MemoryDenyWriteExecute=true
+PrivateTmp=true
+ProtectControlGroups=true
+ProtectHome=true
+ProtectKernelTunables=true
+ProtectProc=invisible
```

Comment:
I *think* this would break the logic that gathers listening sockets on the machine (to populate a WIP "what services are available in your network" UI), but it would manifest as information vanishing rather than errors. Can you leave this out for now and file a bug for me (or you, if you feel so inclined) to check if it's safe to enable?

Reply:
https://github.com/tailscale/tailscale/pull/1062/commits/fdde28942fc623a0b95f6929858250cceb01747a should take care of the first part of this. I’ll try and get to filing the bug in a bit.